### PR TITLE
redesign bmm-page

### PR DESCRIPTION
**Pull request by octobocto**
very open for writing something else in the attempt

Screenshot 2023-11-04 at 12 42 36

### CODE REVIEW

File: lib/pages/tabs/bmm_tab_page.dart [hunk 1]
```diff
@@ -5,189 +5,163 @@ import 'dart:math';
...
+                                "A BMM attempt includes a satoshi denominated 'bid' amount. This amount is paid to "
+                                'the the mainchain miners for the right to construct a new sidechain block. The '
+                                'profit from mining the sidechain block is the total fees collected on the sidechain, '
+                                'minus the bid to the mainchain miners. If the fees collected on the sidechain is '
+                                'less than the bid amount, the profit goes negative (money lost!)'
+                                '',
+                              ),
+                            ),
+                            SailText.primary20(
+                              'No BMM attempts yet. Click the start button!\n(and automine at the same time)',
```
Comment (torkelrogstad) [inline lib/pages/tabs/bmm_tab_page.dart:111]: This assumes that the mainchain is started from the launcher. Could potentially add an option to automine from Sidesail?

Comment (octobocto) [reply lib/pages/tabs/bmm_tab_page.dart:111]: How can we do that?

Comment (octobocto) [reply lib/pages/tabs/bmm_tab_page.dart:111]: I agree that is a very nice UX, but can we hold off a bit on UX benefits like that until everything is implemented?

next iteration kinda stuff

Comment (octobocto) [reply lib/pages/tabs/bmm_tab_page.dart:111]: We should keep adding these UX things that pop up as we go, but keep the scope as small as possible early on imo.

Can keep track of these things in linear, and add them to a later project iteration

**Comment by torkelrogstad** [comment]
On mobile so can't review properly. But from the screen shot: the bid amount is on the mainchain, I.e. regular btc not sbtc. Fees are collected on the sidechain though, so those are sbtc

**Pull request merged**